SalesforceCommerceCloud/pwa-kit
# [Multi-site] Change generator default project for two presets to single-site, single locale

# Description
This PR made changes to the generator to apply config for single site single locale to adopt the changes on #391 

GUS: https://gus.lightning.force.com/a07EE00000cUsEoYAK

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Remove `l10n.config.js` and `pwa-kit.config.js`
- Add `default.json`

# How to Test-Drive This PR

- To test a generated project locally: run `GENERATOR_PRESET=test-project node packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js --outputDir generated-project`
- Run your app after it finishes, your should expect to see `default.json` to have a config for single site, single locale

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
